### PR TITLE
Added documentation for Xdebug 3

### PR DESCRIPTION
**Pull request by sean-e-dietrich**
Added content updates for configuring prior versions of docksal images with xdebug.

### CODE REVIEW

File: docs/content/tools/xdebug.md [hunk 1]
```diff
@@ -190,3 +190,16 @@ To debug Drush commands using Xdebug and VSCode, add the following to your path
     - "Server" can be set to `*`
     - "Server Listen Port" should be set to 9000
     - Make sure "Continue to listen for debug sessions even if the debugger windows are all closed" is checked. This will make the debugger window open automatically.
+
+## Configuring Prior Versions
+
+For versions of XDebug prior to v3.0.0 the following changes will need to be made to the projects `.docksal/docksal.yml`
```
Comment (shelane) [inline docs/content/tools/xdebug.md:196]: ...3.0.0, the following...

**Review by shelane — CHANGES_REQUESTED** [review]

File: docs/content/tools/xdebug.md [hunk 2]
```diff
@@ -188,5 +188,21 @@ To debug Drush commands using Xdebug and VSCode, add the following to your path
...
+
+For versions of XDebug prior to v3.0.0, the following changes will need to be made to the projects `.docksal/docksal.yml`. Additionally, to note the port has changed 
+from 9000 to 9003
+
+```
+services:
+    cli:
+        environment:
+            - XDEBUG_CONFIG=remote_connect_back=0 remote_host=${DOCKSAL_HOST_IP}
+        expose:
```
Comment (sean-e-dietrich) [inline docs/content/tools/xdebug.md:204]: @lmakarov @achekulaev just checking real quick this is valid correct? As we are changing the port in the dockerfile to 9003 the old versions require 9000.

**Comment by sean-e-dietrich** [comment]
Just a note @lmakarov for image versions prior to 2.12 CLI this will introduce a breaking change, but we do have a work around for it.

**Review by shelane — APPROVED** [review]

File: docs/content/tools/xdebug.md (hunk 2 above)
Comment (lmakarov) [reply docs/content/tools/xdebug.md:204]: ✅

**Review by lmakarov — APPROVED** [review]

File: stacks/services.yml [hunk 3]
```diff
@@ -184,9 +184,7 @@ services:
       - DOCROOT
       - VIRTUAL_HOST
       - XDEBUG_ENABLED=${XDEBUG_ENABLED:-0}
-      # TODO: remove xdebug.remote_connect_back in June 2020
-      # See https://github.com/docksal/docksal/issues/1238#issuecomment-566779905
-      - XDEBUG_CONFIG=remote_connect_back=0 remote_host=${DOCKSAL_HOST_IP} # Point xdebug to the listener on the host
+      - XDEBUG_CONFIG=client_host=${DOCKSAL_IP} # Point xdebug to the listener on the host
       - PHP_IDE_CONFIG # This helps with console scripts debugging in PHPStorm (the values is set in docksal-local.env)
```
Comment (shelane) [inline stacks/services.yml:188]: Can you update this:
`- PHP_IDE_CONFIG # This helps with console scripts debugging in PHPStorm (the value is set in docksal-local.env)`

**Pull request merged**